homepage-news.md

```diff
@@ -0,0 +1,149 @@
+# Homepage News Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/config/pages**   
+
+Provide access to the list of all static pages (currently home page news only). These are pages not specific to any items. The design of the REST contract should allow it to be extended to e.g. images as well.
+Because the XMLUI supports home page news per language, multiple pages can be created here.
+
+Example:
+```json
+{
+  "_embedded": {
+    "pages": [
+      {
+        "uuid": "004a297e-fd06-4662-ae51-73e4b7c165c8",
+        "name": "home-page-news",
+        "title": "DSpace Demo Repository",
+        "sizeBytes": 234,
+        "language": "en",
+        "type": "page",
+        "_links": {
+          "content": {
+            "href": "https://dspace7.4science.it/dspace-spring-rest/api/config/pages/004a297e-fd06-4662-ae51-73e4b7c165c8/content"
+          },
+          "format": {
+            "href": "https://dspace7.4science.it/dspace-spring-rest/api/config/pages/004a297e-fd06-4662-ae51-73e4b7c165c8/format"
+          },
+          "languages": {
+            "href": "https://dspace7.4science.it/dspace-spring-rest/api/config/pages/004a297e-fd06-4662-ae51-73e4b7c165c8/languages"
+          },
+          "self": {
+            "href": "https://dspace7.4science.it/dspace-spring-rest/api/config/pages/004a297e-fd06-4662-ae51-73e4b7c165c8"
+          }
+        },
+        "_embedded": {
+          "format": {
```

Comment:
this seems to suggest the use of bitstream in future to store our pages/page fragments/media.
I like this idea but again if we are not going to do that now, I suggest to just model what we have now.